packages/gis/lua/Layer.lua

```diff
@@ -599,6 +622,31 @@ Layer_ = {
 		local project = self.project
 
 		if type(data.layer) == "string" then
+			if string.find(data.layer, "%*") then
+				local base, pattern = extractMultiplesPattern(data.layer)
+				local layers = {}
+				forEachOrderedElement(project.layers, function(layer)
+					table.insert(layers, layer)
+				end)
+
+				layers = findMultiples(base, pattern, layers)
+				forEachOrderedElement(layers, function(_, layer)
+					local attr = data.attribute..layer.pattern
+					if #attr > 10 then
+						customError("The attribute generated '"..attr.."' has more than 10 characters.")
```

Comment:
"The attribute '"..attr.."' to be created has more than 10 characters. Please shorten the attribute name."

Reply:
Note that this error should be applied only when the data is going to be saved into a shapefile. Otherwise it should work properly.